src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptorProvider.cs

```diff
@@ -14,6 +14,8 @@ public class TagHelperDescriptorProvider
     {
         public const string CatchAllDescriptorTarget = "*";
 
+        public static readonly string RequiredAttributeWildcardSuffix = "*";
```

Comment:
Be consistent with `CatchAllDescriptorTarget` or just remove this property. Suggest both should be `public static readonly` since `CatchAllDescriptorTarget` is not pi.